app/services/course/statistics/assessments_score_summary_download_service.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+require 'csv'
+class Course::Statistics::AssessmentsScoreSummaryDownloadService
+  include ApplicationFormattersHelper
+
+  class << self
+    def download(course, assessment_ids)
+      service = new(course, assessment_ids)
+      ActsAsTenant.without_tenant do
+        service.generate_csv_report
+      end
+    end
+  end
+
+  def generate_csv_report
+    assessment_file_name = "#{Pathname.normalize_filename(@course.title)}_assessment_score_summary.csv"
```

Comment:
Why is the filename being generated twice? Can just decalre it once in the higher level job context.

Reply:
refactored to declare it once